Develop Page was created

BoGoBooM opened the pull request:
- [Demo link](https://bogoboom.github.io/layout_KateVR/)

volodymyr-soltys97 reviewed (changes requested):
Good job 👍 
Let's improve your page
Add cursor: pointer and transition for all clickable elements when hovering
<img width="313" alt="image" src="https://github.com/mate-academy/layout_KateVR/assets/64750958/01393fc1-0d16-4433-8706-059b4bc2f881">

Need to remove scroll when this menu is open
<img width="590" alt="image" src="https://github.com/mate-academy/layout_KateVR/assets/64750958/fb694ad7-e562-492d-8c9d-3b097fa46831">

volodymyr-soltys97 reviewed (changes requested):
Still not fixed from previous review
Need to add `cursor: pointer` here
<img width="331" alt="image" src="https://github.com/mate-academy/layout_KateVR/assets/64750958/71766241-11dc-46cb-9925-bcf284c30248">

Also need to remove scroll when this menu is open
<img width="665" alt="image" src="https://github.com/mate-academy/layout_KateVR/assets/64750958/c147a964-25ae-4656-aa4b-22772c44a1aa">

BoGoBooM commented:
I make run delpoy again

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,950 @@
+              </p>
+
+              <div class="touch__link-wrapper">
+                <a href="tel:+86-0571-86105373" class="touch__link">
```
Need to fix format phone
```suggestion
                <a href="tel:+86057186105373" class="touch__link">
```

volodymyr-soltys97 reviewed (changes requested):
Still not fixed from previous review. Need to remove scroll

https://github.com/mate-academy/layout_KateVR/assets/64750958/5f7faec3-d2c4-48f9-ad8e-f99da4791753

BoGoBooM commented:
I'm sorry, but I don't understand where the horizontal scroll on the page came from. I don't have it. I fixed the scroll in the menu. And i fixed line 455
